### ISSUE TITLE
collection.count() missing

### ISSUE THREAD

**Issue by BananaAcid** [mWater/minimongo#36]
I believe it should be included.

**Comment by grassick** [comment]
I agree, though the library meets my needs at this point, so I won't be adding it myself. I am open to pull requests however!
